.github/workflows/ci.yml

```
@@ -8,37 +8,33 @@ on:
 
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'min'
+          - 'lts'
           - '1'
         os:
           - ubuntu-latest
+          - macOS-latest
           - windows-latest
-        arch:
-          - x64
+        exclude:
+          - os: macOS-latest # Apple Silicon
+            version: 'min'
         include:
-          - os: macOS-13
-            arch: x64
-            version: 1
-          - os: macOS-latest
-            arch: aarch64
-            version: 1
-
+          - os: macOS-13 # Intel
+            version: 'min'
```

```suggestion
            version: 'min'
          - os: ubuntu-latest
            version: 'nightly'
```
Might as well keep one test, IMO

---

This was removed in #477 but I'm fine with keeping a test for unreleased Julia versions. Maybe
```suggestion
            version: 'min'
          - os: ubuntu-latest
            version: 'pre'
```
would give a bit more stable and possibly more relevant test results?

---

I'll leave this for a follow-up PR in case we want to test `pre` or `nightly` again.